lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -1,8 +1,7 @@
 const React = require('react');
 
-const title = "This is how you do it.";
-const body = "We are PUMPED to see your photos. Complete these simple steps and upload a photo of yourself in action. Your pics will inspire others to join the movement \u2013 so don't forget to share them!";
-
+const title = "Upload photos of yourself in action!";
+const bodies = ["DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"];
```

Comment:
`bodies` sounds weird. Could we rename to `content` maybe and then in the loop refer to it as `paragraph` or something...? Nit-picky. Whatever you change it to, change it in the **CampaignSlide.js** & **ReportbackItemsSlide.js** too.

Just a thought...